niworkflows/data/nipreps.json

```
@@ -95,31 +95,31 @@
     },
     {
       "name": "atlas",
-      "pattern": "[_/\\\\]+atlas-([a-zA-Z0-9]+)"
+      "pattern": "[_/\\\\]+atlas-([a-zA-Z0-9+]+)"
     },
     {
       "name": "roi",
-      "pattern": "[_/\\\\]+roi-([a-zA-Z0-9]+)"
+      "pattern": "[_/\\\\]+roi-([a-zA-Z0-9+]+)"
     },
     {
       "name": "label",
-      "pattern": "[_/\\\\]+label-([a-zA-Z0-9]+)"
+      "pattern": "[_/\\\\]+label-([a-zA-Z0-9+]+)"
     },
     {
       "name": "fmapid",
-      "pattern": "[_/\\\\]+fmapid-([a-zA-Z0-9]+)"
+      "pattern": "[_/\\\\]+fmapid-([a-zA-Z0-9+]+)"
     },
     {
       "name": "desc",
-      "pattern": "[_/\\\\]+desc-([a-zA-Z0-9]+)"
+      "pattern": "[_/\\\\]+desc-([a-zA-Z0-9+]+)"
     },
     {
       "name": "from",
-      "pattern": "(?:^|_)from-([a-zA-Z0-9\\+]+).*xfm"
+      "pattern": "(?:^|_)from-([a-zA-Z0-9+\\+]+).*xfm"
```

`to` `from` and `space` already handled the `+` character, though we can clean these up to simply

```suggestion
      "pattern": "(?:^|_)from-([a-zA-Z0-9+]+).*xfm"
```

---

Sorry about that! I just did a find and replace. Since this was already merged I can open a PR to fix this.